### PR TITLE
feat: sketches user profile

### PR DESCRIPTION
**Pull request by taylorichigo**

### CODE REVIEW

File: components/tabsUserProfile/tabFollowers.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```
Comment (pSchwietzer) [inline components/tabsUserProfile/tabFollowers.tsx:1]: Redundant React Import

File: backend/graphQLSchema.ts [hunk 2]
```diff
@@ -1,30 +0,0 @@
...
-})
-
-schemaBuilder.queryType()
-schemaBuilder.mutationType()
-
-buildMetaSchema()
-buildIngredientSchema()
-buildRecipeSchema()
-
-export default schemaBuilder.toSchema()
```
Comment (lonba) [inline backend/graphQLSchema.ts:30]: Why is this deleted?

File: components/tabsUserProfile/input.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.tabProfileInput{
```
Comment (Bhoft) [inline components/tabsUserProfile/input.css]: remove css

**Review by moncapitaine — APPROVED** [review]
